tests/PHPStan/Rules/Classes/data/trait-instance-of.php

```diff
@@ -0,0 +1,25 @@
+<?php declare(strict_types = 1);
+
+namespace TraitInstanceOf;
+
+trait Foo {
+	public function test(HelloWorld $hw): string {
+		if ($hw instanceof HelloWorld) {
```

Comment:
since I had different implementations in between which also swalled errors for lokal vars within traits, I added a separate test-case for it